data/product.d/fedora-eln.conf

```diff
@@ -1,8 +1,8 @@
 # Anaconda configuration file for Fedora ELN.
 
 [Product]
-product_name = Fedora
-variant_name = ELN
+product_name = Fedora-ELN
+variant_name = Everything
```

Comment:
The variant name doesn't have to be specified when it is not necessary.

Reply:
How does one determine whether it's necessary?

Reply:
It's not necessary because you want to have this configuration for all the variants.